src/components/Menu/Menu.tsx

```diff
@@ -69,8 +69,9 @@ export const Menu = ({
           className={menu}
           defaultOpenKeys={defaultOpenKeys}
           defaultSelectedKeys={defaultSelectedKey ? [defaultSelectedKey] : undefined}
+          getPopupContainer={() => document.querySelector(`.${menu}`)!}
```

Comment:
The style is applied correctly also without this prop. Are you sure it is necessary?

Reply:
It is needed for nested menus to inherit CSS properties in the vertical mode. I inserted an explanatory comment ✅